examples/morpheusvm/tests/e2e/e2e_test.go

```diff
@@ -47,6 +47,7 @@ var _ = ginkgo.SynchronizedBeforeSuite(func() []byte {
 
 	// Run only once in the first ginkgo process
 	nodes := tmpnet.NewNodesOrPanic(flagVars.NodeCount())
+	nodes[0].Flags["http-port"] = "9650"
```

Comment:
Could we re-package this set of functions into a single helper, which sets up the network and aliases the chain as well, so we don't need to perform aliasing in `Ping` ?